src/runtime/substitution.cc

```diff
@@ -841,7 +841,8 @@ bool GraphXfer::create_new_operator(OpX const *opx, Node &op) {
     case OP_CONCAT: {
       int axis;
       assert(opx->get_pm_constraint(PM_AXIS, axis));
-      op = model->get_or_create_concat_node(opx->inputs.size(), inputs, axis);
+      std::vector<ParallelTensor> _inputs(std::begin(inputs), std::end(inputs));
+      op = model->get_or_create_node<Concat>(_inputs, {axis});
```

Comment:
```suggestion
      op = model->get_or_create_node<Concat>({std::begin(inputs), std::end(inputs)}, {axis});
```
Also, `std::end` works on constant sized arrays? Wasn't aware of this